http://sourceware.org/ml/gdb-cvs/2012-04/msg00130.html

### src/gdb/doc/ChangeLog	2012/04/17 15:56:21	1.1297
### src/gdb/doc/ChangeLog	2012/04/18 07:03:57	1.1298
## -1,3 +1,7 @@
+2012-04-18  Jan Kratochvil  <jan.kratochvil@redhat.com>
+
+	* gdb.texinfo (Auto-loading verbose mode): Fix smallexample typo.
+
 2012-04-17  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	New option "set debug auto-load".
--- src/gdb/doc/gdb.texinfo	2012/04/17 15:56:21	1.944
+++ src/gdb/doc/gdb.texinfo	2012/04/18 07:03:58	1.945
@@ -21192,7 +21192,7 @@
 may not be too obvious while setting it up.
 
 @smallexample
-(gdb) set debug auto-load ues
+(gdb) set debug auto-load on
 (gdb) file ~/src/t/true
 auto-load: Loading canned sequences of commands script "/tmp/true-gdb.gdb"
            for objfile "/tmp/true".
